src/store/movie/operations/getMovieCreditsOperation.js

```diff
@@ -0,0 +1,34 @@
+import { createLogic } from 'redux-logic'
+import { normalize } from 'normalizr'
+import { uniq } from 'lodash'
+
+import { movieCredits } from 'Schemas'
+import { storeData } from 'Store/data/actions'
+import { movieSelector } from 'Store/movie/selectors'
+import { GET_MOVIE_CREDITS } from '../types'
+import {
+  requestMovieCreditsStart,
+  requestMovieCreditsSuccess,
+  requestMovieCreditsError
+} from '../actions'
+
+export const getMovieCreditsOperation = createLogic({
+  type: GET_MOVIE_CREDITS,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { movieId } = action
+    dispatch(requestMovieCreditsStart())
+    try {
+      const response = await axios.get(`/movie/${movieId}/credits`)
+      const { movie, persons } = normalize(response.data, movieCredits).entities
+      const storeMovie = movieSelector(getState(), movieId)
+      const { cast, crew } = movie[movieId]
+      dispatch(storeData('persons', persons))
+      dispatch(storeData('movies', { [movieId]: { ...storeMovie, cast, crew: uniq(crew) } }))
```

Comment:
why do we need to call `uniq` here?

Reply:
@VladWithCookies because the same person can be crew or cast member

Reply:
@VladWithCookies In case the same person has few jobs or characters in the same movie, the response will have few person objects with the same ID. That’s why I added the merge strategy

Reply:
can we call uniq inside merge strategy of the schema?

Reply:
@VladWithCookies  As I see, no, we can't. I don’t have access to the `result` in `mergeStrategy` or `processStrategy`